daras_ai_v2/text_to_speech_settings_widgets.py

```
@@ -325,32 +345,44 @@ def azure_tts_voices() -> dict[str, dict[str, str]]:
     return {voice.get("ShortName", "Unknown"): voice for voice in res.json()}
 
 
-def bark_selector():
+def bark_selector(*, language_filter: str = ""):
+    voice_options = list(BARK_ALLOWED_PROMPTS)
+    if language_filter:
+        voice_options = [None] + filter_models_by_language(
+            language_filter,
+            {key: [key.split("_")[0]] for key in voice_options if key is not None},
+        )
```

Ruff (0.15.18)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)